src/Workspaces/Core/Portable/LanguageServices/SemanticsFactsService/ISemanticFactsService.cs

```diff
@@ -101,6 +101,8 @@ internal interface ISemanticFactsService : ILanguageService
 
         IEnumerable<ISymbol> GetDeclaredSymbols(SemanticModel semanticModel, SyntaxNode memberDeclaration, CancellationToken cancellationToken);
 
+        IParameterSymbol FindParameterForArgument(SemanticModel semanticModel, SyntaxNode argumentNode, CancellationToken cancellationToken);
```

Comment:
I didn't understand what this API has to do with the embedded languages feature.

Reply:
mentioned over in CSharpSemanticFactsService.cs:  

> added to help out the regex/json features (which want to see if they're used in situatiosn like JToken.Parse(...) and new Regex(...)).

basically, both of them end up needing to go ask "is this a string being passed as an argument to a certain parameter?".  So i just pulled that helper into this PR since it was trivial to do so.